builder/.bin/bin.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+const path = require('path');
+
+const CONFIG_FILE_NAME = '30s.config.json';
+
+require('yargs')
```

Comment:
I suggest to bring some order here, move `require` on top next to other requires and move `getConfig` before commands parsing to make structure more linear